source/docs/software/labview/resources/how-to-use-joystick-buttons-to-control-motors-or-solenoids.rst

```diff
@@ -0,0 +1,77 @@
+How to Use Joystick Buttons to Control Motors or Solenoids
+==========================================================
+
+.. image:: images/ni-logo.png
+
+As we all get our drive systems working, we are moving on to connecting our auxiliary devices such as motors and solenoids.  With this, we will generally use joystick buttons to control these devices.  To get started with this, we'll go through several ways to control devices with joystick buttons.
+
+Did you know that you can click and drag a VI Snippet from a document like this right into your LabVIEW code?  Try it with the snippets in this document.
+
+Setup:
+------
+
+No matter what the configuration, you'll need to add one, two, or more (if you're really excited) joysticks to the "Begin.vi".  The first example uses 2 joysticks and the others only use one.  Give each one a unique name so we can use it in other places, like the snippet below.  I named them "LeftStick" and "RightStick" because they are on the left and right sides of my desk.  If your joysticks are already configured, great! You can skip this step.
+
+.. image:: images/how-to-use-joystick-buttons-to-control-motors-or-solenoids/setup.png
+
+The rest of the code in this document will be placed in the "Teleop.VI" This is where we will be programming our joystick buttons to control different aspects of our motors or solenoids.
+
+Scenario 1
+----------
+
+I want a motor to move one way when I press one button and the other way when I press a different button.
```

Comment:
Could you surround the scenarios in quotation marks to add emphasis?

```suggestion
"I want a motor to move one way when I press one button and the other way when I press a different button."
```

Reply:
Sure. I think bold might also make it clearer.
```suggestion
**"I want a motor to move one way when I press one button and the other way when I press a different button."**
```